project/api/models.py

```diff
@@ -182,3 +183,55 @@ def clean(self, *args, **kwargs):  # pylint: disable=W0221
     def save(self, *args, **kwargs):  # pylint: disable=W0221
         self.full_clean()
         super(SpacetimeVolume, self).save(*args, **kwargs)
+
+class Narrative(models.Model):
+    """
+    Stores narrative information.
+    """
+
+    author = models.CharField(max_length=100)
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    start_date = models.CharField(max_length=100)
+    end_date = models.CharField(max_length=100)
```

Comment:
@MiklerGM do we need this? Can we take `[earlist_map_datetime, latest_map_datetime]`?

Reply:
Sure we can take it from `narrations.year`